build.gradle.kts

```diff
@@ -1,11 +1,9 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 plugins {
-    `java-library`
     `maven-publish`
     signing
-    kotlin("jvm") version "1.7.20"
+    kotlin("multiplatform") version "1.7.20"
     kotlin("plugin.serialization") version "1.7.20"
+    id("org.jetbrains.dokka") version "1.7.20"
```

Comment:
Dokka setup looks good. `javadoc-silence.txt` is not necessary although Dokka is [quite noisy](https://github.com/Kotlin/dokka/issues/1894).